### PR TITLE
rnote: new, 0.11.0

### PR DESCRIPTION
**Pull request by chiyuki0325**
Topic Description
-----------------

- rnote: new, 0.11.0

Package(s) Affected
-------------------

- rnote

Build Order
-----------

```
#buildit rnote
```

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [ ] AMD64 `amd64`
- [ ] AArch64 `arm64`
- [ ] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

Architectural progress for secondary ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-creativity/rnote/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=rnote
+PKGDES="Sketch and take handwritten notes."
```
Comment (MingcongBai) [inline app-creativity/rnote/autobuild/defines]: Please make this more descriptive: what is it and what does it do? You have answered the latter but not the former.

Also, on styling: do not include a period point (.) at the end.

File: app-creativity/rnote/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+PKGSEC=graphics
```
Comment (MingcongBai) [inline app-creativity/rnote/autobuild/defines]: Pretty sure this should be `doc`.

File: app-creativity/rnote/spec [hunk 3]
```diff
@@ -0,0 +1,5 @@
+VER=0.11.0
+SRCS="tbl::https://github.com/flxzt/rnote/archive/v$VER.tar.gz"
```
Comment (MingcongBai) [inline app-creativity/rnote/spec]: Please use a Git tag per the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/), we do not allow using platform-generated tarballs as the checksums may not be stable over time.

File: app-creativity/rnote/spec [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+CHKSUMS="sha256::b133d4331963d3c09d3a7477f60fc4c5072471dcbf459379a593ca1724164af4"
+
+CHKUPDATE="github::repo=flxzt/rnote"
```
Comment (MingcongBai) [inline app-creativity/rnote/spec]: Please use `anitya::` (I know, it's down right now, but let's try again in a few hours).

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-doc/rnote/autobuild/defines [hunk 5]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=rnote
+PKGDES="Vector-based drawing app for sketching and handwritten notes"
+PKGSEC=doc
+PKGDEP="gtk-4 glib libadwaita poppler gstreamer alsa-lib"
+BUILDDEP="meson cargo llvm cmake git gtk-update-icon-cache"
```
Comment (xtexx) [inline app-doc/rnote/autobuild/defines]: Please remove `meson cargo git` per style guidelines.

Comment (MingcongBai) [reply app-doc/rnote/autobuild/defines]: Actually, replace `cargo` with `rustc` as it is the actual package behind `cargo`.

**Review by MingcongBai — DISMISSED** [review]

**Review by purofle — APPROVED** [review]

**Pull request merged**